docs/xxportablesharedcache.md

```diff
@@ -0,0 +1,38 @@
+<!--
+* Copyright (c) 2020, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]PortableSharedCache
+
+This feature enables AOT compiled code to be generated based on a chosen set of processor features (features present in the Intel SandyBridge processors) that maximizes its portability across machines. It is currently supported only on x86. The feature is turned on by default in Docker containers that can be disabled with "-XX:-PortableSharedCache". The portable shared cache feature state is persisted in the SCC which means specifying it with an existing SCC has no effect.
```

Comment:
Intel SandyBridge -- Is this now Intel Celeron? (Or am I just confused!) Trademarks required:  Intel&reg; Celeron&reg; (I don't think SandyBridge would need one.)
`-XX:-PortableSharedCache` -- pleas make monospace with back ticks as in table.
SCC - on first occurrence, please spell out: "...shared classes cache (SCC)..."

Reply:
Actually I think it should be Intel Sandy Bridge (https://en.wikipedia.org/wiki/Sandy_Bridge).